assets/search/main.ejs

```diff
@@ -16,16 +16,16 @@
       <div class="row">
         <div class="col-md-12">
           <ul class="list-group search-results-adv">
-            <li class="list-group-item"
-                ng-if="search.inProgress">
-              Searching...
-            </li>
             <a ng-repeat="result in search.results track by $index"
                ng-href="{{search.linkAddress(result)}}"
                ng-if="result !== null"
                class="list-group-item">
               <%- partial(searchType + '/result.ejs') %>
             </a>
+            <li class="list-group-item"
+                ng-if="search.inProgress">
+              Searching...
+            </li>
```

Comment:
Another unrelated fix: I moved the 'Searching' message to be below the results (in the same place as the 'get more results' button), rather than above. Now the results don't jump down when you try to get more.